src/StatsModels.jl

```diff
@@ -2,33 +2,64 @@ __precompile__(true)
 
 module StatsModels
 
-using Compat
-using DataFrames
+using Tables
 using StatsBase
-using Compat.SparseArrays
-using Compat.LinearAlgebra
-using Compat: @debug, @warn
+using CategoricalArrays
+using DataStructures
 
-export @formula,
-       Formula,
-       ModelFrame,
-       ModelMatrix,
+using SparseArrays
+using LinearAlgebra
+using Missings
 
-       AbstractContrasts,
-       EffectsCoding,
-       DummyCoding,
-       HelmertCoding,
-       ContrastsCoding,
+using Tables: ColumnTable
 
-       coefnames,
-       dropterm,
-       setcontrasts!
+export
+    #re-export from StatsBase:
+    StatisticalModel,
+    RegressionModel,
+    
+    @formula,
+    ModelFrame,
+    ModelMatrix,
 
+    AbstractContrasts,
+    EffectsCoding,
+    DummyCoding,
+    HelmertCoding,
+    ContrastsCoding,
+
+    coefnames,
+    dropterm,
+    setcontrasts!,
+
+    AbstractTerm,
+    TermOrTerms,
```

Comment:
Does this have to be exported?

Reply:
Nope.

Reply:
Umm, I did use it in MixedModels (src/randomeffectsterm.jl) but I can qualify the name